owncloudApp/src/main/java/com/owncloud/android/authentication/oauth/OAuthConnectionBuilder.kt

```diff
@@ -28,8 +30,9 @@ class OAuthConnectionBuilder(val context: Context) : ConnectionBuilder {
      */
 
     @Throws(IOException::class)
-    override fun openConnection(uri: Uri): HttpURLConnection {
+    override fun openConnection(uri: Uri): HttpConnection {
         val conn: HttpURLConnection
+        val conWrapper: HttpConnection
```

Comment:
I don't see any use of conWrapper, is it needed?

Reply:
OAuthConnectionBuilder is not required anymore the whole class should be deleted.